aspnetcore/fundamentals/minimal-apis/includes/parameter-binding8.md

```diff
@@ -200,11 +200,11 @@ Binding from form-based parameters using <xref:Microsoft.AspNetCore.Http.IFormCo
 
 The following code uploads files using inferred binding from the `IFormFile` type:
 
-:::code language="csharp" source="~/fundamentals/minimal-apis/parameter-binding/samples8/Iform/Program.cs" highlight="17-22,43-57":::
+:::code language="csharp" source="~/fundamentals/minimal-apis/parameter-binding/samples8/Iform/Program.cs" highlight="18-23,44-50":::
 
 ***Warning:*** When implementing forms, the app ***must prevent*** [Cross-Site Request Forgery (XSRF/CSRF) attacks](xref:security/anti-request-forgery?view=aspnetcore-8.0&preserve-view=true#afwma). In the preceding code, the <xref:Microsoft.AspNetCore.Antiforgery.IAntiforgery> service is used to prevent XSRF attacks by generating and validation an anti-forgery token:
 
-:::code language="csharp" source="~/fundamentals/minimal-apis/parameter-binding/samples8/Iform/Program.cs" highlight="24,44":::
+:::code language="csharp" source="~/fundamentals/minimal-apis/parameter-binding/samples8/Iform/Program.cs" highlight="25,45":::
```

Comment:
Fixes the highlights:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/73969091-2f9e-4925-8f5b-c6ff0f95d369)